src/PhpSpreadsheet/Cell/Cell.php

```diff
@@ -71,6 +71,18 @@ class Cell
      */
     private $formulaAttributes;
 
+    /** @var bool */
```

Comment:
I might move these (and the corresponding individual getters/setters) into a separate class, and just have a single getter/setter in Cell for `get/setIgnoreErrors()`

Reply:
I'm not sure what you have in mind here. Do you want the user to do something like:
```php
$cell->setIgnoreErrors('numberStoredAsText'); // obviously provide constants
```
Where the code in Cell does something like:
```php
private $ignoreErrors = new IgnoreErrors();

public function setIgnoreErrors(string $type, bool $flag)
{
    $this->ignoreErrors->apply($type, $flag);
}
```

Reply:
I was thinking more like
```
$cell->getIgnoreErrors()->setignoredErrorFormula(true);
```
doesn't even need a setter in the Cell, just one new method there for the getter; then in the Cell\IgnoreErrors class, a getter/setter for each of the error types that can be ignored, defaulting to false unless set by the Reader, or by a user.
It keeps the increase in the number of new methods/properties in Cell down to a minimum; but still gives explicit methods for each error type that can be ignored

Reply:
Got it - thanks. Latest push implements this suggestion.